<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartCalc</class>
 <widget class="QMainWindow" name="SmartCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>249</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1098</width>
    <height>249</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1098</width>
    <height>249</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartCalc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(255, 255, 255);
background-color: rgb(220, 220, 220);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
border-radius:5px;
border-style: solid;
border-width: 5px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
background-color: rgb(87, 106, 96);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PlotButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>GRAH</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>531</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xStart">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>220</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xEnd">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yEnd">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yStart">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CreditButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>76</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>CREDIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DepositButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>76</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>DEPOSIT</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>103</y>
      <width>475</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="SinButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="AtanButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>atan</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="TanButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="LnButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="OneButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="MulButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4" colspan="2">
      <widget class="QPushButton" name="ZeroButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="PlusButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="SevenButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="PowButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="NineButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="MinusButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="AcButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(255, 38, 0) ;
}</string>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="LogButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="EightButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="Div1Button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="CosButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="AsinButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>asin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="SixButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="ResultButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="BracketLeftButton">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="AcosButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>acos</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="FiveButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="Div2Button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="TwoButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}


</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="XButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="FourButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="SqrtButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="ThreeButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="BracketRightButton">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 2px;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0) ;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="DotButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius:2px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
background-color:rgb(0, 0, 0) ;
}</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>электроника</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>МК 85М</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditX">
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>70</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Xlabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>27</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X = </string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
